src/System.CommandLine/Argument.cs

```diff
@@ -10,7 +10,7 @@
 namespace System.CommandLine
 {
     /// <summary>
-    /// Represents a value passed to an <see cref="Option"/> or <see cref="Command"/>.
+    /// A symbol defining a value that can be passed to a <see cref="ICommand">command</see> or <see cref="IOption">option</see>.
```

Comment:
nit: Is "passed" really the right word when used in relation to the the `ICommand` and `IOption`? I think passed is a little ambiguous since it is not clear if that is passed in code or passed on the command line. Perhaps something more like: "A symbol defining a value that can be passed on the command line to a command or option."

Reply:
Added "on the command line". 

Do you think that the `see` tags make this more ambiguous?

Reply:
No i think those are good